web/impact/impact/tests/test_user_detail_view.py

```diff
@@ -662,7 +662,6 @@ def test_get_user_with_no_created_at_in_prgs_handled(self):
         role_grant = ProgramRoleGrantFactory(
             program_role__program__program_status=ACTIVE_PROGRAM_STATUS,
             program_role__program__program_family__name="Quigville",
```

Comment:
no need to specify a program_family__name here, please remove

Reply:
Done